### ISSUE TITLE
A separate server address for SFTP in node condig

### ISSUE THREAD

**Issue by Doc94** [pterodactyl/panel#2483]
**Is your feature request related to a problem? Please describe.**
Actually for the conection of nodes i use a subdomain in cloudflare, this subdomain are in proxy, then only work with SSL in for HTTP requests, in the case of SFTP details show the same subdomain but not work in conections for the proxy.

**Describe the solution you'd like**
A optional variable for sftp server address in any node. in cases where the node is in proxy then can set a sftp connection (like ftp.domain.com or the IP).

**Comment by Homie20006** [comment]
I can totally relate to this.
Exact same setup and users can't use the visible node address for SFTP.

This extra option for a custom address would simply resolve the problem. 👍

**Comment by elitetheespeon** [comment]
Adding to this: I have a 1:1 NAT setup with internal IPs assigned to external IPs and while I can alias the game servers IP addresses, SFTP still shows the local address in the panel to users. Being able to customize this as an IP or FQDN would be helpful.

**Comment by TekExplorer** [comment]
What if its akin to the aliases for ip:port allocations? or would it be better to have that specified in the node config?

**Comment by Doc94** [comment]
> What if its akin to the aliases for ip:port allocations? or would it be better to have that specified in the node config?

Delay...

aliases for ip:port allocations not think can be great.. because for example in my case i use the aliases for set a SRV DNS for the game and this not work with the SFTP. its better a aliase for SFTP connection in node.

**Comment by maxlingenfelter** [comment]
This would be a very helpful feature. Currently, the only way to have a proxied node but allow sftp that I know of is to modify the panel's code to point to a different place for sftp.

**Comment by BIackHornet** [comment]
This is a needed feature for those using free Cloudflare proxy services. Please give us an alternate SFTP hostname field that will change the web links and perhaps work with the transfer server feature.

**Comment by ghost** [comment]
Agreed this feature is needed, especially with some reverse proxies where the SFTP address is already on a different port.

**Comment by tomlyo** [comment]
Just wanted to chime in to say that this is still very relevant, I'm using Cloudflare as well as many others in here who could benefit from specifying an SFTP URL.

**Comment by maxlingenfelter** [comment]
> Just wanted to chime in to say that this is still very relevant, I'm using Cloudflare as well as many others in here who could benefit from specifying an SFTP URL.

I completely agree and could really use this feature!

**Comment by tschlienger** [comment]
I'd love to see this feature implemented! I'm running the admin panel behind a proxy which only support HTTP(S) protocols, so I published the SFTP service using a different domain, which makes the address shown in the admin panel and the address unusable.

**Comment by sidboy55555** [comment]
Any news on this? I hope it gets implemented.

**Comment by Loki-101** [comment]
Agreed. This feature would be amazing.

**Comment by TekExplorer** [comment]
Please stick to adding a 👍 to the issue instead of sending unconstructive comments. (i get it, you want it, but its not helpful)
Also remember that you can subscribe to this issue to get updates on its progress

**Comment by Mutex21** [comment]
> I'd love to see this feature implemented! I'm running the admin panel behind a proxy which only support HTTP(S) protocols, so I published the SFTP service using a different domain, which makes the address shown in the admin panel and the address unusable.

Or via enterprise cloudflare which is not cheap. I had love to see too this feature to be implemented.

**Comment by anondev-sudo** [comment]
This would be good for those who use cloudflare to proxy wings

**Comment by Greaper88** [comment]
I just came across having this same issue. My setup has Cloudflare mainly for the flexible cert, and then I have a NGINX reverse proxy for the individual nodes. I finally got the https part to work, but I have this same issue where server transfer won't work. I can use the SFTP if I use the root address from WinSCP though, but the panel isn't capable of doing that as of yet.

**Comment by Northwave** [comment]
As many really would have a big issue solved by this feature, please consider adding it :) Also same would apply for the shown bind IP in the server consoles, as this also dont work with cloudflare and reverse proxys.

**Comment by WardPearce** [comment]
agreed, sftp alias would be useful in cases on using a proxy like cloudflare. I dont mind if people I allow access to my game panel know my public Ip but i dont want the IP to be completely public

**Comment by TheekshanaA** [comment]
+1 This would be a great addition!

**Comment by knurhp** [comment]
or let us manually override the sftp domain settings from the panel tab

**Comment by DLandDS** [comment]
I'd very glad to see this feature implemented. i really need this.

**Comment by laasth** [comment]
I'm giving this feature a thumbs up 👍 Really needed

**Comment by ytcalifax** [comment]
This issue has been open for 3+ years, is there a chance this will be ever implemented?

**Comment by lukariosmen** [comment]
👍

**Comment by ririko5834** [comment]
Can you please implement this feature into official pterodactyl panel?

**Comment by Inrixia** [comment]
+1

**Comment by Ry6000** [comment]
This would still be an awesome feature to have!
